e2e/test/scenarios/native/native-database-source.cy.spec.js

```diff
@@ -0,0 +1,247 @@
+import { SAMPLE_DB_ID } from "e2e/support/cypress_data";
+import {
+  restore,
+  popover,
+  addPostgresDatabase,
+  POPOVER_ELEMENT,
+} from "e2e/support/helpers";
+
+const PG_DB_ID = 2;
```

Comment:
Isn't this `WRITABLE_DB_ID` ?

Reply:
In theory yes.
But it can be misleading. We're using `WRITABLE_DB_ID` only in the context of actual writable scenarios, and we're not writing anything to this database.

I'm working on a better solution for all QA DB references, btw.